usabilla-react-native/README.md

```diff
@@ -40,12 +40,11 @@ react-native link usabilla-react-native
 To use this Bridge, please make sure you are using XCode 9.1 or above.
 
 ## Features
-The first version of Usabilla for React Native `0.1.0` allows you to :
 - Load and show a Passive Feedback form.
 - Submit the results of the form.
+- Pre-fill a Passive Feedback form with a custom screenshot.
```

Comment:
can we elaborate more on this feature as we did for the others and add an example code call .

Reply:
Yes we will do that in its own [subtask](https://usabilla.atlassian.net/browse/SDK-1015) :)